### ISSUE TITLE
Accessibility: Cannot open the System Tray Menu with Keyboard

### ISSUE THREAD

**Issue by chigkim** [OpenVPN/openvpn-gui#763]
I'm a screen reader user on Windows 10, and taking the following steps doesn't trigger to open the system tray menu.

1. Press  windows+d to go to desktop
2. Press tab until you get to the system tray
3. Press up/down to select OpenVPN GUI
4. Press either the application key or shift+f10 to open the menu

It looks like it has been broken since 2.6.10-I001-amd64.
It works with 2.6.9-I001-amd64.
I'd really appreciate if you Could investigate and fix the issue.
Thanks!

**Comment by selvanair** [comment]
I can reproduce this. Seems we broke this in PR #670  where a custom tooltip window was introduced.

I think the reason is that we always used WM_RBUTTONUP in  OnNotifyTray() instead of WM_CONTEXTMENU. Unsure why this shows up only when the tooltip was customized.

**Comment by selvanair** [comment]
@chigkim A fix has been merged. Should be in next 2.6 and 2.7 releases.
